ingestion/src/metadata/profiler/interface/sqlalchemy/profiler_interface.py

```diff
@@ -537,6 +534,9 @@ def get_all_metrics(
                     logger.debug(traceback.format_exc())
                     logger.error(f"Operation was cancelled due to TimeoutError - {exc}")
                     raise concurrent.futures.TimeoutError
+                except KeyboardInterrupt:
+                    pool.shutdown(wait=True, cancel_futures=True)
```

Comment:
I believe you'll need to use `shutdown39` to have it compatible with Python 3.8.

Reply:
changed to shutdown39